packages/react-native-reanimated/android/build.gradle

```diff
@@ -349,63 +349,63 @@ android {
     sourceSets.main {
         java {
             if (IS_NEW_ARCHITECTURE_ENABLED) {
-                srcDirs += "src/fabric/java"
+                srcDirs += "src/fabric/reanimated/java"
```

Comment:
I don't think we need `reanimated/` here since we're about to introduce a new separate `build.gradle` for worklets